pkg/trace/agent/agent_test.go

```diff
@@ -308,6 +308,35 @@ func TestProcess(t *testing.T) {
 		assert.Equal(t, "tracer-hostname", tp.Hostname)
 	})
 
+	t.Run("inferred spans", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("skipGlobalTags", func(t *testing.T) {
```
We only use one "word" everywhere when naming tests. No spaces.